client/Windows/wf_defaults.c

```diff
@@ -0,0 +1,118 @@
+/**
+ * FreeRDP: A Remote Desktop Protocol Implementation
+ *
+ * Copyright 2014 Marc-Andre Moreau <marcandre.moreau@gmail.com>
+ * Copyright 2022 Stefan Koell
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <Windows.h>
+#include <wincred.h>
+#include <stdio.h>
+#include <malloc.h>
+#include <freerdp/settings.h>
+
+static PCWSTR ValidateString(const BYTE* pb, ULONG cb)
+{
+	if (!pb || !cb)
+		return 0;
+
+	if (cb & (sizeof(WCHAR) - 1))
+		return 0;
+
+	if (*(WCHAR*)(pb + cb - sizeof(WCHAR)))
+		return 0;
+
+	return (PCWSTR)pb;
+}
+
+static void AddDefaultSettings_I(rdpSettings* settings, size_t idHostname, size_t idUsername,
+                                 size_t idPassword)
+{
+	static const PSTR TERMSRV = "TERMSRV/%s";
+
+	PSTR TargetName = 0;
```

Comment:
While actually the same, `PSTR` is a pointer type, so better initialize with `NULL` (not blocking, just a remark)

Reply:
`PSTR TargetName = NULL;` (`NULL` is for pointers, value is (most of the time) same as 0)